React-Native/TodoList/src/app/index.js

```diff
@@ -1,15 +1,15 @@
 import React, { Component } from 'react';
-import Main from './screens/Main';
 import { Provider } from 'react-redux';
 import store from '../redux/store';
+import AppContainer from './AppContainer';
 
 // import '../config';
 
 export default class App extends Component {
```

Comment:
make the export at the end. You could make a function here.. not need of a class